src/test/regress/expected/gporca_optimizer.out

```diff
@@ -13729,7 +13729,7 @@ and first_id in (select first_id from mat_w);
                      ->  Shared Scan (share slice:id 1:0)  (cost=0.00..431.00 rows=4 width=4)
                            Output: share0_ref2.first_id
  Optimizer: Pivotal Optimizer (GPORCA)
- Settings: enable_seqscan=on, optimizer=on
+ Settings: optimizer=on
 (31 rows)
```

Comment:
Why not change the above test case to verify current patch, e.g. replace the type of `second_id` from integer to some more complex (timestamp in your provided case)?

Reply:
This test was introduced to show a different issue. Yes, this issue is related to ours, but it's different, so I don't think it's correct to modify existed test.
The test related to our issue was introduced by this patch separately.